core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -80,16 +80,16 @@ private[spark] class ExecutorAllocationManager(
     Integer.MAX_VALUE)
 
   // How long there must be backlogged tasks for before an addition is triggered (seconds)
-  private val schedulerBacklogTimeout = conf.getLong(
-    "spark.dynamicAllocation.schedulerBacklogTimeout", 5)
+  private val schedulerBacklogTimeoutS = Utils.timeStringAsS(conf.get(
```

Comment:
very minor, but in general I find it weird for variables to have a capital letter at the end. I would call this either `schedulerBacklogTimeoutSeconds` or just leave it as before. Not a big deal if you don't change this.